<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTicketDialog</class>
 <widget class="QDialog" name="CreateTicketDialog">
  <property name="windowTitle">
   <string>Создание тикета</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelProject">
     <property name="text">
      <string>Проект*</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="projectCombo"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelTracker">
     <property name="text">
      <string>Трекер*</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="trackerCombo"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>Статус*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="statusCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelPriority">
     <property name="text">
      <string>Приоритет*</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="priorityCombo"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelAssignee">
     <property name="text">
      <string>Назначена*</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="assigneeCombo"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelWatcher">
     <property name="text">
      <string>Наблюдатель*</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="watcherCombo"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelStartDate">
     <property name="text">
      <string>Дата начала*</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDateEdit" name="startDateEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Название*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="titleEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelTags">
     <property name="text">
      <string>Теги</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="tagEdit"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPlainTextEdit" name="descriptionEdit"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelFiles">
     <property name="text">
      <string>Файлы</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QVBoxLayout" name="attachedFilesLayout"/>
   </item>
   <item row="12" column="0" colspan="2">
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="attachBtn">
       <property name="text">
        <string>Прикрепить файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
